src/main/resources/templates/xfer-validate.html

```diff
@@ -0,0 +1,99 @@
+		
+		<div th:replace="common/menus :: rp-breadcrumbs"/>
+    
+    	<!-- Page Content Begin -->
+    
+	    	<div class="content mt-3">
+	            <div class="animated fadeIn">
+	                
+	              
+	               
+	            	<div class="row">
+                  		<div class="col-lg-12">
+                    		<div class="card">
+                        		<div class="card-header bg-flat-color-1">
+                            		<strong class="text-white card-title">VISA Direct Payement</strong>
+                        		</div>
+
+               <br>         		
+ 										<div class="row form-group">
+		    							<div class="col col-sm-4">
+					                            <label class=" form-control-label"><strong>Visa Account Number</strong></label>
+				                            	<select class="form-control" id="visaAccount" name="visaAccount" required>
+				                            		<option class="text-light"value="">---------- Select Account -----------</option>
+				                              		<option value ="ABCD1234ABCD123">Account Ending in 123</option>
+				                              		<option value ="ABCD1234ABCD456">Account Ending in 456</option>
+				            					</select>
+					                    </div>
+					                   	</div>
+		                        		<div class="row form-group">
+		                        			<div class="col col-sm-4">
+					                            <label class="form-control-label"><strong>Transfer Amount</strong></label>
+					                            <input type="text" class="form-control" th:field="${transaction.amount}" required th:pattern="${patternTransactionAmount}">
+					                        	<small class="form-text text-muted">ex. $25.00</small>
+					                        </div>
+		                        		</div>	
+	                        		
+		                        		<button type="button" onclick="loadDoc()">Submit</button>
+		                        		<h1> Action Code </h1>
+		                        		<h1 id="actioncode"> if id="actioncode" == 57 (Approved) </h1>
+		                        		<h1> Approval Code </h1>
+		                        		<h1 id="approvalCode"></h1>
+		                        		<h1 id="selectedaccount"></h1>
+		                        		
+
+
+<script>
+function loadDoc() {
+  	var selectedaccount = "visaAccount"
+	var xhttp = new XMLHttpRequest();
+  xhttp.onreadystatechange = function() {
+    if (this.readyState == 4 && this.status == 200) {
+    	var result = JSON.parse(this.responseText);
+    	this.responseText = 'json';
+    	Data = result.actionCode; // or json["Data"
+    	document.getElementById("actioncode").innerHTML = result.actionCode ;
+    	document.getElementById("approvalCode").innerHTML = result.approvalCode;
+     }
+  };
+
+  xhttp.open("POST", "https://paymentaccounts76-8080-ns-bzctjenkins.crane.blazemeter.net/visadirect/fundstransfer/v1/pullfundstransactions", true);
```

Comment:
Shouldn't this be env-driven? Otherwise how can I override this endpoint to point to my own mock service for demos?